backend/src/main/java/de/iav/backend/controller/TimeSlotController.java

```diff
@@ -0,0 +1,81 @@
+package de.iav.backend.controller;
+
+import de.iav.backend.model.TimeSlotResponseDTO;
+import de.iav.backend.model.TimeSlotWithoutIdDTO;
+import de.iav.backend.service.TimeSlotService;
+import jakarta.validation.Valid;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/api/toastMasterManager/timeslots")
```

Comment:
Konvention ist folgende:

```suggestion
@RequestMapping("/api/toast-master-manager/timeslots")
```

Reply:
Umgesetzt